DataManagementSystem/Agent/RequestOperations/RemoveFile.py

```diff
@@ -55,7 +55,7 @@ def __init__( self, operation = None, csPath = None ):
     gMonitor.registerActivity( "RemoveFileFail", "Failed file removals",
                                "RequestExecutingAgent", "Files/min", gMonitor.OP_SUM )
     # # re pattern for not existing files
-    self.reNotExisting = re.compile( "(no|not) such file or directory", re.IGNORECASE )
+    self.reNotExisting = re.compile( "(no|not) such file.*", re.IGNORECASE )
```

Comment:
Ok, I'll change it anyway to be sure.
In any case, since we are about to change the propagation of errors... ;-)